### DISCUSSION TITLE
En qué parte en la base de datos queda registrado el propertylevelconfig

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#169] [2. Questions and Answers] [answered]
Hola

Revisamos con @jdurandpaz un tema del ordenamiento de los canales mediante una propiedad.
Desde el web admin, sabemos que ir a grupo / editar / activar colección adicional y seleccionar la colección.

Desde la base de datos, no vemos ese valor y nos entró la duda: En dónde estará o es un bug?

Adjunto SS

![image](https://user-images.githubusercontent.com/61419471/117827020-a0c65100-b23e-11eb-868d-3ff001e92e9c.png)

![image](https://user-images.githubusercontent.com/61419471/117827033-a4f26e80-b23e-11eb-8833-bcc04bbd02ec.png)

Gracias :)

**Comment by acedward** [comment, marked as answer]
Por favor, no utlizar la base de datos directamente.
Si hay un error por favor reportar.

**Reply by NicolasPLG** [reply]
@acedward no ocupamos la base de datos para ordenar la colección, solo es una consulta en dónde queda el elemento, porque no lo vemos en la base y podría ser un bug o un feature.

:)

**Reply by rafaelwalter** [reply]
@NicolasPLG cambiamos donde se almacena la configuracion, ya no se guarda en `propertyLevelsConfig`.
Funciona el ordenamiento con lo que configurarón en el admin?
Si no ha funcionado avisanos y crearemos un ticket interno! por favor no mirar la base de datos.

**Reply by jdurandpaz** [reply]
@rafaelwalter en el admin no funciona, pero es el cliente con infra dedicada. El error es /{"message":"'/actions' property does not exist\n" 

Finalmente lo pude agregar por base usando  "groupBy" : ObjectId("60817597fa2dd30008250d9b").

**Reply by gmoplaza** [reply]
@rafaelwalter lo conversamos con @jdurandpaz y como el dice se puso solucionar el problema y entiendo que ya levantó el ticket. Parece ser un problema con Core que no permite cambiar el atributo actions

**Reply by jdurandpaz** [reply]
si muchas gracias @gmoplaza ya que justo nos pidieron revisar eso en la reu !!

**Reply by rafaelwalter** [reply]
ok! gracias @gmoplaza. @NicolasPLG por favor cerrar discussion
